ConsoleApp/ConsoleApp/Program.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace ConsoleApp
+{
+
+ 
+
+
+    class Program
+    {
+
+        static void todayHoliday(IEnumerable<Holiday> holidays)
+        {
+            var sb = new StringBuilder();
+
+
+            DateTime currentDate = (DateTime.Now).Date;
+
+            var resultNext = holidays
+            .Where(x =>DateTime.Compare(currentDate, Convert.ToDateTime(x.date)) < 0)
+                .Select(x=>new
+                { 
+                  x.date,
+                  x.nameEn
+                })
+                .OrderBy(x => x.date) 
+                .First();
```

Comment:
it's possible that today e.g. 1st of January and First() will fail with exception
the same below for 31st of December